src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -626,6 +1105,44 @@ public static void Test1(int x)
             Assert.Equal("System.Double <anonymous type: System.Int32 aa, System.String $1, System.Double bb>.bb { get; }", properties[2].ToTestDisplayString());
         }
 
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/20338")]
+        public void AnonymousTypeSymbols_Error_DuplicateName_OperationTree()
+        {
+            string source = @"
+class ClassA
+{
+    public static void Test1(int x)
+    {
+        object v1 = /*<bind>*/new
+        {
+            aa = 1,
+            ClassA.aa,
+            bb = 1.2
+        }/*</bind>*/;
+    }
+
+    public static string aa = ""-field-aa-"";
+}
+";
+            string expectedOperationTree = @"
+IAnonymousObjectCreationExpression (OperationKind.AnonymousObjectCreationExpression, Type: <anonymous type: System.Int32 aa, System.String $1, System.Double bb>, IsInvalid) (Syntax: 'new ... }')
+  Initializers(3): IAssignmentExpression (OperationKind.AssignmentExpression, Type: System.Int32, Constant: 1) (Syntax: 'aa = 1')
+      Left: IPropertyReferenceExpression: System.Int32 <anonymous type: System.Int32 aa, System.String $1, System.Double bb>.aa { get; } (Static) (OperationKind.PropertyReferenceExpression, Type: System.Int32) (Syntax: 'aa')
+      Right: ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1) (Syntax: '1')
+    IAssignmentExpression (OperationKind.AssignmentExpression, Type: System.Double) (Syntax: 'ClassA.aa')
+      Left: IPropertyReferenceExpression: System.Double <anonymous type: System.Int32 aa, System.String $1, System.Double bb>.bb { get; } (Static) (OperationKind.PropertyReferenceExpression, Type: System.Double) (Syntax: 'bb')
+      Right: IFieldReferenceExpression: System.String ClassA.aa (Static) (OperationKind.FieldReferenceExpression, Type: System.String) (Syntax: 'ClassA.aa')
+    ILiteralExpression (Text: 1.2) (OperationKind.LiteralExpression, Type: System.Double, Constant: 1.2) (Syntax: '1.2')
+";
```

Comment:
I may be misreading the expected result, but it looks like the string represents `{ aa = 1, bb = ClassA.aa, 1.2 }`.